Add `Vocaloid Mapping Contest 2 & New Featured Artist: iroha(sasaki)` news post and associated wiki article (#10847)
ppy/osu-wiki: wiki/Contests/VMC/2/en.md

---
tags:
  - VMC2
  - VMC
---

# Vocaloid Mapping Contest 2

![VMC banner](/wiki/Contests/VMC/2/img/bannervmc.jpg)

> The **Vocaloid Mapping Contest** (***VMC***) is a beatmapping contest hosted by [Vocaloid Appreciation Team (VAT)](https://discord.gg/EruvDHaReY), and is all about promoting VOCALOID, UTAU, and other vocal synth songs and artists that have fallen through the cracks. Mappers were tasked with creating the best beatmap possible, and then having them judged by some of the most prominent VOCALOID mappers in the community. 

Reviewer: ```suggestion
The **Vocaloid Mapping Contest** (***VMC***) is a beatmapping contest hosted by the Vocaloid Appreciation Team, and is all about promoting VOCALOID, UTAU, and other vocal synth songs and artists that have fallen through the cracks. Mappers were tasked with creating the best beatmap possible, and then having them judged by some of the most prominent VOCALOID mappers in the community. 
```

This second iteration of the VMC featured a new VOCALOID Featured Artist, support for every game mode, and the return of a legendary VOCALOID mapper as a guest judge.

## Contest schedule

| Event | Timestamp |
| --: | :-- |
| Announcement | 2024-01-27 |
| Submission | 2024-01-27 to 2024-03-01 |
| Screening\* | 2024-03-01 to 2024-03-08 |
| Judging | 2024-03-08 to 2024-03-29 |
| Results livestream | 2024-03-31 @ 18:00 UTC (tentative) |

> The results are streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Details regarding the stream were provided in the newspost and on the Discord channel.

Reviewer: ```suggestion
The results are streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Further details regarding the stream will be provided in this article, the news post, and in the Discord server.
```

\**Prize only for participants of the osu! game mode, due to it's one-of-a-kind nature.*

All participating mappers receive constructive feedback from experienced mappers in the community.

## Links

- [Announcement news post](https://osu.ppy.sh/home/news/)
- [Contest listing](https://osu.ppy.sh/community/contests/)
- [Discord server](https://discord.gg/EruvDHaReY)

## Ruleset

- **Submissions must use one of the [selected songs](#songs).**
- **Submissions must be in `.osz` format.**
- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes (such as an accidental off-screen object), submissions that contain intentional unrankables (not due to simple error, such as sliderator) will be disqualified. Mappers will be notified if this occurs.
- **Submissions must not be publicly revealed until after the results are announced.**
- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges and screeners.
- **Collaborations are not allowed.** This is a solo contest.
- **Submissions must not include storyboards or videos.**

> - **Your submission must not have any edits to the `.osu` file that would not be possible through the editor.** This isn't Aspire. You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.

Reviewer: ```suggestion
- **Your submission must not have any edits to the `.osu` file that would not be possible through the editor.** This is not Aspire. You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
```

- **Your submission must have only one difficulty.**

## Judges 

> | Gamemode | Judge Names |

Reviewer: ```suggestion

| Game mode | Judges |
```

> | :-: | :-: |

Reviewer: ```suggestion
| :-: | :-- |
```

| Creativity | 25 | How unique and outstanding is the map and the elements within it compared to the current mapping meta? How well does it stand out from the crowd? Does it evoke a feeling of "wow!", "that's cool!", or "that's super interesting!"? Generic mapping that copies the current mapping meta or concepts from other famous maps is discouraged. |
| Expertise | 20 | This refers to how well the map executes the elements it presents, encompassing the technical skills of the mapper, as well as the map's consistency, structure, and style. Judges will score this entry considering the overarching picture of the map, and will avoid nitpicking small details. |
| Overall Impression | 20 | The VOCALOID mappers participating as judges for this contest will all have personal biases with regard to how these songs are mapped. The judges will provide comments regarding the score they give. |
| Hitsounding | 5 | Rankable hitsounds can give up to 2 points, but hitsounds that fundamentally add to the experience and complement the map in a unique way can give the full 5 points. |

## Songs

> Five (5) songs were selected by the staff from those provided by iroha(sasaki) for the purpose of this mapping contest.  These songs were selected based on several factors, such as whether they have been ranked in the past, their suitability for the purposes of contest mapping, and other factors.

Reviewer: ```suggestion
Five (5) songs were selected by the staff from those provided by iroha(sasaki) for the purpose of this mapping contest. These songs were selected based on several factors, such as whether they have been ranked in the past, their suitability for the purposes of contest mapping, and other factors.
```
